freeclimb/__init__.py

```diff
@@ -15,7 +15,7 @@
 
 from __future__ import absolute_import
 
-__version__ = "2.2.0"
+__version__ = "3.0.0"
```

Comment:
I think SS are a minor change as they're reverse compatible

Reply:
I don't think we should make these changes a minor change in the Python SDK, while they are major change in the Java SDK. It could cause confusion.
https://github.com/FreeClimbAPI/java-sdk/pull/13